neworigins/html/neworigins.html

```diff
@@ -0,0 +1,7004 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
```

Comment:
The atlantis rules are supposed to be auto-generated (so that they are accurate about all the game settings), so probably shouldn't be in the repository.

Reply:
good catch